crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -323,9 +323,14 @@ def post_cleanup_repos
           false
         )
 
-        Crowbar::Repository.chef_data_bag_destroy(
-          "#{repository.data_bag_name}/#{repository.id}"
-        )
+        begin
+          Crowbar::Repository.chef_data_bag_destroy(
+            "#{repository.data_bag_name}/#{repository.id}"
+          )
+        rescue Net::HTTPServerException
```

Comment:
we should, but we can't. first we have to catch all `Net::HTTPServerException` and then if `404` is not in the error response code, we have to raise an Exception again in the rescue block.

I could change it, would that be fine with you? but then it would not be consistent anymore with all the other places where we handle databags, but we could make a follow up.